initializers/tenants_groups.yml

```diff
@@ -0,0 +1,4 @@
+#- name: Tenant Group 1
```

Comment:
The name of this file is wrong; it should be `tenant_groups.yml`.

Reply:
Fixed.